examples/arn_generic.py

```diff
@@ -206,36 +207,36 @@ def main(main_params):
 			print("train-set accuracy at epoch %d: %f" % ((1 + epoch), lnsaccuracy))
 			performance['lnsacc_train'][epoch] = 100 * lnsaccuracy
 			lnscorrect_count = 0  #do same # as in train set for val set
-			for mbatch in range(int(split / batchsize)):
+			for mbatch in range(int(val_split / batchsize)):
 				start = mbatch * batchsize
 				x = x_val[start:(start + batchsize)]
 				y = y_val[start:(start + batchsize)]
 				if main_params['type'] == 'xlnsnp':
-				  lnsx = xl.xlnsnp(np.array(xl.xlnscopy(np.array(x,dtype=np.float64))))
+					lnsx = xl.xlnsnp(np.array(xl.xlnscopy(np.array(x,dtype=np.float64))))
 				if main_params['type'] == 'xlnsnpv':
-				  lnsx = xl.xlnsnpv(np.array(xl.xlnscopy(np.array(x,dtype=np.float64))))
+					lnsx = xl.xlnsnpv(np.array(xl.xlnscopy(np.array(x,dtype=np.float64))))
 				if main_params['type'] == 'xlnsnpb':
-				  lnsx = xl.xlnsnpb(np.array(xl.xlnscopy(np.array(x,dtype=np.float64))),2**2**-xl.xlnsF)
+					lnsx = xl.xlnsnpb(np.array(xl.xlnscopy(np.array(x,dtype=np.float64))),2**2**-xl.xlnsF)
 				if main_params['type'] == 'xlns':
-				  lnsx = (np.array(xl.xlnscopy(np.array(x,dtype=np.float64))))
+					lnsx = (np.array(xl.xlnscopy(np.array(x,dtype=np.float64))))
 				if main_params['type'] == 'xlnsud':
-				  lnsx = (np.array(xl.xlnscopy(np.array(x,dtype=np.float64),xl.xlnsud)))
+					lnsx = (np.array(xl.xlnscopy(np.array(x,dtype=np.float64),xl.xlnsud)))
 				if main_params['type'] == 'xlnsv':
-				  lnsx = (np.array(xl.xlnscopy(np.array(x,dtype=np.float64),xl.xlnsv)))
+					lnsx = (np.array(xl.xlnscopy(np.array(x,dtype=np.float64),xl.xlnsv)))
 				if main_params['type'] == 'xlnsb':
-				  lnsx = (np.array(xl.xlnscopy(np.array(x,dtype=np.float64),xl.xlnsv,2**2**-xl.xlnsF)))

```

Comment:
Lots of line changes like this? Can you avoid these and re-create a pull request?